src/strong-type.ts

```diff
@@ -7,11 +7,12 @@ export interface StrongType<ValueT> {
 	getNull: () => ValueT | null;
 	reset: () => void;
 	typeId: 'StrongType' | string;
+	_data: STData<ValueT>;
 }
 
 export function makeStrong<ValueT>(
 	fallbackArg: ValueT,
-	initial?: ValueT | null,
+	initial: ValueT | null | undefined,
```

Comment:
- [ ] Change `| undefined` type on `initial` to use the optional operator instead `?`.